Assets/Scripts/Cards/Spells/Spell.cs

```diff
@@ -54,37 +55,48 @@ public bool ValidateConditions()
             return this.ConditionData.Check(ParentSpell.Result);
         }
 
-        public override void ExecuteAction()
+        public override async void ExecuteAction()
```

Comment:
We shouldn't use Async here, only in the FX classes. The EntityActions and buffer isn't made to handle async this way